### PR TITLE
Improvements in TF for perf tests

### PR DESCRIPTION
**Pull request by imjfckm**

### CODE REVIEW

File: test_utils/os_utils.py [hunk 1]
```diff
@@ -142,6 +142,25 @@ def get_dut_cpu_number():
...
 
+def get_dut_cpu_physical_cores():
+    """ Get list of CPU numbers that don't share physical cores """
+    output = TestRun.executor.run_expect_success("lscpu --all --parse").stdout
+
+    core_list = []
+    visited_phys_cores = []
+    for line in output.split("\n"):
+        line = line.split("#")[0].rstrip()
+        if len(line) == 0:
```
Comment (Ostrokrzew) [inline test_utils/os_utils.py:153]: Wouldn't
``` python
    if '#' in line:
        continue
```
be easier?

File: test_tools/fio/fio_param.py [hunk 2]
```diff
@@ -303,6 +303,11 @@ def add_job(self, job_name=None):
         self.fio.jobs.append(new_job)
         return new_job
 
+    def clear_jobs(self):
+        self.fio.jobs = []
+
+        return self
```
Comment (Ostrokrzew) [inline test_tools/fio/fio_param.py:309]: Is this `return` necessary?

Comment (imjfckm) [reply test_tools/fio/fio_param.py:309]: It's the convention within `FioParam` methods to enable chaining of methods

File: test_utils/os_utils.py (hunk 1 above)
Comment (imjfckm) [reply test_utils/os_utils.py]: It would, fixed!

**Pull request merged**